napari/components/_viewer/view/main.py

```diff
@@ -73,7 +76,42 @@ def __init__(self, viewer):
                 'standard': QCursor()
             }
 
-    def set_cursor(self, cursor, size=10):
+        self.viewer.events.interactive.connect(self._on_interactive)
+        self.viewer.events.cursor.connect(self._on_cursor)
+        self.viewer.events.reset_view.connect(self._on_reset_view)
+        self.viewer.layers.events.reordered.connect(self._update_canvas)
+
+    def screenshot(self, region=None, size=None, bgcolor=None):
```

Comment:
What's the point of moving this here if no one is going to be able to use it?

Reply:
I think this goes back to what @royerloic was hinting at during yesterday's meeting, that we need a controller class that combines the model and the given view (depending on backend) while also providing convenience functionality for the user.

Reply:
for now I forwarded the `screenshot` method inside the `Viewer` object so it's still accessible. Agreed a controller might be needed. Also now it's a little strange that `to_svg` is still on `ViewerModel`, but `screenshot` is on `QtViewer`. Maybe we can fix in future PRs though

Reply:
`screenshot` is also able to be called from `Viewer`, there are no issues with `self`

Reply:
yes, when you first told me this i thought you had forwarded it on the class level but the instance level works fine